uberfire-project/uberfire-project-api/src/main/java/org/guvnor/common/services/project/categories/Model.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.guvnor.common.services.project.categories;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import org.uberfire.workbench.category.Category;
+
+@ApplicationScoped
+public class Model extends Category {
```

Comment:
The name Model sounds too generic. Java/Data/Process/Form some other model? Could you make the category name a bit more specific?

Reply:
The category names were previously selected. And the resource type for those categories too. I can  give you the document where they are voted